### DISCUSSION TITLE
DetailedList "will squeeze other components inside the same box out of the screen in iOS environment, but it works normally in macOS environment.

### DISCUSSION THREAD

**Discussion by jlij** [beeware/toga#2828] [Debugging help]
DetailedList "will squeeze other components inside the same box out of the screen in iOS environment, but it works normally in macOS environment.

**Comment by freakboy3742** [comment]
You're going to need to provide some more details. Examples of code, screenshots - *anything* that explains what you mean, what you expect to see, and what you're actually seeing.

**Comment by jlij** [comment]
![截屏2024-09-11 15 06 44](https://github.com/user-attachments/assets/b1c2378c-258e-4e69-8dc3-54f6731c9431)
![截屏2024-09-11 15 28 27](https://github.com/user-attachments/assets/52f34617-79e7-4ab8-badd-578d3b7974f4)

**Reply by jlij** [reply]
The button below is squeezed out of the screen.

**Reply by freakboy3742** [reply]
It would also be very helpful to know the code you're using to generate this. It's impossible to debug a problem if we don't know the code that is associated with a behavior.

**Reply by jlij** [reply]
```
import toga
from toga.style import Pack
from toga import Box, DetailedList, Button
from toga.style.pack import COLUMN, ROW





class testoga(toga.App):
    def startup(self):
        my_dict = [{"icon":None,'title':'title','subtitle':'subtitle'}]

        self.passenger_dl = DetailedList(data=my_dict, style=Pack(flex=1))

        self.record_button = [Button(text=i,  style=Pack(flex=1, font_size=18)) for i in  ['A', 'B', 'C', 'D']]
        self.button_box = Box(children=self.record_button, style=Pack(direction=ROW))

        self.page_box = Box(children=[ self.passenger_dl, self.button_box],
                              style=Pack(flex=1, direction=COLUMN))

        self.main_window = toga.MainWindow()
        self.main_window.content =self.page_box
        self.main_window.show()


def main():
    return testoga()
```

**Reply by jlij** [reply]
briefcase dev
![截屏2024-09-13 08 04 00](https://github.com/user-attachments/assets/493c9168-608b-4f62-94d8-f50326338ca4)
briefcase run iOS
![截屏2024-09-13 08 05 43](https://github.com/user-attachments/assets/be8716ed-15b8-48c7-86b2-f0ad64de6c3c)

**Reply by jlij** [reply]
The button below is squeezed out of the screen

**Reply by freakboy3742** [reply]
Looks like you've found a bug in the recent support adding simple windows. I've added #2836 to track the problem.
